### ISSUE TITLE
Watch error: the filename, directory name, or volume label syntax incorrect

### ISSUE THREAD

**Issue by dy** [rsms/estrella#15]
Getting the following error for watch mode:
```
PS C:\projects\xxx> estrella -w index.js -o dist.js     
error: Failed to write to output file: open C:\projects\xxx\.esbuild.?.meta.json: The filename, directory name, or volume label syntax is incorrect.
1 error
Watching files for changes...
estrella.js: TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at i (C:\Users\dfcre\AppData\Roaming\nvm\v14.3.0\node_modules\estrella\<estrella>\watch\watch.ts:55:29)
    at Object.nl (C:\Users\dfcre\AppData\Roaming\nvm\v14.3.0\node_modules\estrella\<estrella>\watch\watch.ts:99:3)
    at rn (C:\Users\dfcre\AppData\Roaming\nvm\v14.3.0\node_modules\estrella\<estrella>\estrella.js:768:23)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

**Comment by dy** [comment]
Hey... Any updates? Having to run chokidar-cli for now...

**Comment by jimrandomh** [comment]
`?` is an invalid character on NTFS. Looking at `estrella.js`, it looks like that `?` is coming from `config.projectID`, which comes from https://github.com/rsms/estrella/blob/bd3050e6c2897c5f71b7fa1eb96f212f598f037c/src/config.ts . That default should probably be changed from `"?"` to something filename-safe, but also you may be able to make it work by changing your config in a way that makes this something non-default.

**Comment by rsms** [comment]
Fix will be included in the next release (1.3.4)

**Comment by rsms** [comment]
Shipped with [release 1.4.0](https://github.com/rsms/estrella/releases/tag/v1.4.0)
